EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -855,7 +851,7 @@ void HcalUnpacker::unpack(const FEDRawData& raw, const HcalElectronicsMap& emap,
       }
     }
   }
-}
+}      
```

Comment:
These whitespace changes may cause problems with other PRs that are also changing this file. Please eliminate the unnecessary differences on lines 605, 606, 650, and 854.